tools/sst_dump_tool.cc

```diff
@@ -55,6 +55,8 @@ void print_help(bool to_stderr) {
 
     --command=check|scan|raw|verify|identify
         check: Iterate over entries in files but don't print anything except if an error is encountered (default command)
+               When read_num, from and to are not set, it compares the number of keys read with num_entries in table
```

Comment:
read_num -> read_num_limit

Reply:
This part is used in the CLI so I did not include the change in this PR.  We can change them separately.